dts/msm8226/msm8926-samsung-kmini3g.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <0xC808FF01 2 0x10001>;
```

Comment:
This already corresponds to msm8926-samsung-r02.dts

Please move your kmini3g node there.

Reply:
How?